saltgui/static/scripts/panels/Minions.js

```diff
@@ -25,7 +25,7 @@ export class MinionsPanel extends Panel {
     this.addWarningField();
     this.addTable(["Minion", "Status", "Salt version", "OS version", "-menu-"]);
     this.setTableSortable("Minion", "asc");
-    this.setTableClickable();
+    this.setTableClickable("cmd");
```

Comment:
This page cannot manage ALT or CRTL on a row click. I would remove the info command.

_PS: I assume this is the home page panel, and this code adds the info reference. If this is not the case, ignore this comment or apply it to the correct file._

Reply:
thx, I will look into it once I am back at my dev environment...

FYI
The "Minions" page is the default homepage.
Any other page can be configured to be the homepage.
See also https://github.com/erwindon/SaltGUI/#reduced-menus.

Reply:
the function `setTableClickable` has 3 possible parameters:
* `cmd`: when clicking on a row shows the "CMD" popup (and no special handling of CTRL/ALT)
* `page`: when clicking on a row navigates to a new page and considers CTRL/ALT to do that in a new tab
* (none): when clicking on a row does nothing at all.

the presence of a parameter-value (i.e. the first 2 cases) causes the info-icon to be shown in to top-right corner of the table. the information shown there is different for each case.

so, the original code was correct.

Reply:
I verified each occurrence of `setTableClickable` and found no unintended behavior.

The only odd one is the rows from page "Nodegroups" (only visible when you actually have nodegroups defined).
All regular rows properly behave with "cmd". But this table also shows minions that are not in "accepted" state. For these rows, the click is ignored and when not ignored should act using "page". I decided earlier that it better to not do that.

Reply:
Thanks for the extra details!